static/src/javascripts/projects/common/modules/commercial/permutive.js

```diff
@@ -19,6 +19,12 @@ const getSegments = (key: string): Array<string> => {
 export const getPermutiveSegments = () => getSegments(PERMUTIVE_KEY);
 export const getPermutivePFPSegments = () => getSegments(PERMUTIVE_PFP_KEY);
 
+export const clearPermutiveSegments = () => {
+    storage.local.remove(PERMUTIVE_KEY);
+    storage.local.remove(PERMUTIVE_PFP_KEY);
+    return [];
```

Comment:
I find the `return []` a bit confusing here, could it be better to clear them inside the `getSegments` method which we will expect it to return en empty array instead?

Reply:
Not sure why it's confusing? The `getSegments` method also returns an empty array if it fails.
Happy to remove it.

Reply:
Just because `clearPermutiveSegments` does 2 things
- Clears entries from localStorage
- Returns an empty array
I think it would make sense to move the ternary operation logic inside getPermutiveSegments or just create a new wrapper function?
`getSegments` looks like it's responsible to return the segments back even if it's an empty array.

Just my view when I was reading it but feel free leave it as is :)